packages/grafana-ui/src/components/TagsInput/TagItem.tsx

```diff
@@ -29,8 +29,8 @@ export const TagItem = ({ name, disabled, onRemove }: Props) => {
         size="lg"
         disabled={disabled}
         ariaLabel={`Remove "${name}" tag`}
+        tooltip={`Remove tag`}
         onClick={() => onRemove(name)}
-        type="button"
```

Comment:
@L-M-K-B think this means that the button now defaults to `type="submit"`... which means if it's used in a form it will submit the form. i'm struggling to think if there's any case where we'd want an `IconButton` to submit a form, so maybe the safest thing to do would be to add `type="button"` to `IconButton` itself?